content/applications/sales/sales/send_quotations/different_addresses.rst

```diff
@@ -2,74 +2,54 @@
 Deliver and invoice to different addresses
 ==========================================
 
-With Odoo Sales, you can configure different addresses for delivery and invoicing. For some
-customers, it will be very practical to define specific billing and shipping addresses. Indeed, not
-everyone will have the same delivery location as the invoicing location.
+With Odoo Sales, customers can use different addresses for delivery and invoicing.
 
 Activate the feature
 ====================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Customer Addresses**
-feature.
+Activate the :guilabel:`Customer Addresses` feature in the settings. It is located in
+:menuselection:`Accounting > Configuration > Settings` if the Accounting app is installed, or
+:menuselection:`Invoicing > Configuration > Settings` if the Accounting app is not installed.
 
-.. image:: different_addresses/addresses_1.png
+.. image:: different_addresses/customer-addresses-setting.png
    :align: center
    :class: img-thumbnail
-   :alt: How to enable different customer addresses on Odoo Sales?
+   :alt: Activate the Customer Addresses setting
 
-Add addresses from a quotation
-==============================
+Configure the contact form
+==========================
 
-When you create a quotation, you must add a customer. This customer can be a company or a person
-with specific billing and shipping addresses already defined and registered in the system, or it can
-be a new customer. In this case, you have to **Create and edit** the contact form for your new
-customer and link it, if necessary, to a company. In this contact form, you will be able to add,
-delete and modify invoice and delivery addresses.
+Navigate to the :guilabel:`Contacts` application, or to :menuselection:`Sales > Orders >
+Customers`, and click on a customer to open their contact form. Under the :guilabel:`Contacts &
+Addresses` tab, click :guilabel:`Add`.
```

Comment:
Navigate to the :guilabel:`Contacts` application (or to :menuselection:`Sales > Orders >
Customers`), and click on a customer to open their contact form. 

Under the :guilabel:`Contacts & Addresses` tab, click :guilabel:`Add`.